frontend/public/components/dashboard/project-dashboard/details-card.tsx

```diff
@@ -18,6 +18,7 @@ export const DetailsCard: React.FC = () => {
   const labelsSubset = _.take(keys, 3);
   const firstThreelabels = _.pick(obj.metadata.labels, labelsSubset);
   const detailsLink = `${resourcePathFromModel(ProjectModel, obj.metadata.name)}/details`;
+  const serviceMeshEnabled = obj.metadata.labels['maistra.io/member-of'] === 'istio-system';
```

Comment:
Would it be better to move this `istio-sytem` to a shared `const` as it is checked in multiple places?